scripts/watch_containers.py

```diff
@@ -38,9 +44,9 @@ def watch_containers(name, since=datetime.utcnow(), polling_time=DEFAULT_POLLING
                 successful_containers.append(container)
         except docker.errors.APIError:
             deleted_containers.append(container)
-    finish_time = datetime.utcnow() - since
+    finish_time = datetime.utcnow() - init_time
     finish_time_in_minutes = finish_time / timedelta(minutes=1)
-    message = "Finished. Total {} -- Successful {} -- Failed {} -- Deleted {}. It tooks {} minutes".format(len(containers), len(successful_containers), len(failed_containers), len(deleted_containers), int(finish_time_in_minutes))
+    message = "Finished build. Total {} -- Successful {} -- Failed {} -- Deleted {}.\nIt tooks {} minutes".format(len(containers), len(successful_containers), len(failed_containers), len(deleted_containers), int(finish_time_in_minutes))
```

Comment:
Minor typo: should be `It took {} minutes`

Reply:
I was doubting about it hhehe. Thx